### PR TITLE
mapillary API V4

### PR DESCRIPTION
**Pull request by astomas**
new mapillary photo id retrieval with new mapillary API (2022)

### CODE REVIEW

File: StreetViewButtons.js [hunk 1]
```diff
@@ -132,9 +144,22 @@ L.StreetView = L.Control.extend({
       last = b;
 
       var tmpl = b._template;
+	  var latitude1 = center.lat - 0.0002;
```
Comment (Zverik) [inline StreetViewButtons.js]: This block is not used for mapillary (which sets `needUrl` to false) — please remove it, for it looks like a hack.

**Review by Zverik — CHANGES_REQUESTED** [review]

File: StreetViewButtons.js (hunk 1 above)
Comment (astomas) [reply StreetViewButtons.js:147]: sorry i dont understand which line(s) to remove. Can you do it yourself if you have rights? i'm not used to work on github

Comment (astomas) [reply StreetViewButtons.js]: is that ok now?

Comment (astomas) [reply StreetViewButtons.js]: do you have rights to edit my fork?

Comment (astomas) [reply StreetViewButtons.js]: if I remove (delete) l.144, 146, var tmpl is not defined anymore

Comment (Zverik) [reply StreetViewButtons.js]: Done, I've added a few commits to clean up.

**Comment by astomas** [comment]
how will you fill the 4 bbox coord l.22?

**Comment by astomas** [comment]
hi (back from holiday)
can I close the pull request?
At the moment the code wont work cause you need 4 coord
(lat1,lat2,lon1,lon2) to insert into mapillary new url call

Le jeu. 28 juil. 2022 à 14:30, Ilya Zverev ***@***.***> a
écrit :

**Pull request closed without merging**